VARIABLES.md

```diff
@@ -35,6 +35,7 @@
 | enable_gluesync | Enable metadata sync from Hive to the Glue catalog. | bool | `false` | no |
 | enable_hive_metastore_metrics | Enable sending Hive Metastore metrics to CloudWatch. | bool | `false` | no |
 | enable_metadata_events | Enable Hive Metastore SNS listener. | bool | `false` | no |
+| enable_ranger_logs | Enable storing Ranger Audit logs. | bool | 'false' | no |
```

Comment:
I really think that we should just use `ranger_policy_manager_url` as the criteria.   E.g.:
```
          env {
            name  = "ENABLE_RANGER_LOGS"
            value = var.ranger_policy_manager_url != "" ? "1" : ""
          }
```
etc.

Reply:
I'll try this and will update PR when I can make it work correctly